catalog/CatalogRelation.cpp

```diff
@@ -93,9 +93,9 @@ CatalogRelation::CatalogRelation(const serialization::CatalogRelation &proto)
     }
   }
 
-  // Deserializing the indices of the relation
-  for (int i = 0; i < proto.index_size(); i++) {
-    indices_.push_back(proto.index(i));
+  // Deserializing the index scheme defined for the relation, if any
```

Comment:
Add the period symbol at the end of the comment to treat it as an English sentence.